### DISCUSSION TITLE
Disable output tab drag and drop

### DISCUSSION THREAD

**Discussion by Sathiskumar18** [eclipse-theia/theia#9493] [Q&A] [answered]
Seems the Output window is draggable which makes collapse the other components in the window.
Is there any option available to disable the output tab drag and drop or required any new implementation to do that?
Thanks.

**Comment by vince-fugnitto** [comment, marked as answer]
The question is a duplicate of https://community.theia-ide.org/t/how-to-block-theia-output-dockpanel-widget-drag-to-other-panel/1691.
